javascript/sdk/lib/controllers/iam.ts

```diff
@@ -76,16 +76,38 @@ export default class IAMController {
 
   /** Create a new user inside an account */
   async createUser(
```

Comment:
This should be a new function called something like `verifyUser` or `verifyToken` which does a GET to `/iam/user` and passes a token to `?token={token}`. This should use `this.#client.request("/iam/user"` so we can call it without being logged in

Reply:
so what else needs to be in the new function? Just token and thats all?

Reply:
@khyberspache